config/routes.rb

```diff
@@ -32,6 +32,8 @@
   resources :users, except: [:index]
   resources :activities, only: [:index]
 
+  patch "/organisation_session" => "organisation_session#update"
```

Comment:
How would you feel about giving this a bit of name spacing, maybe `/users/session/organisation` (only because that is where all the Devise routes are)?